### ISSUE TITLE
Configuration 'KoverEngineConfig' was resolved during configuration time

### ISSUE THREAD

**Issue by JiongBull** [Kotlin/kotlinx-kover#235]
hi there, after i upgrade AGP to 7.3 and gradle to 7.4.2 today, there is a new warning about kover, i have no idea how to resolve it.

```shell
gradle koverMergedReport
Configuration 'KoverEngineConfig' was resolved during configuration time.
This is a build performance and scalability issue.
See https://github.com/gradle/gradle/issues/2298
Run with --info for a stacktrace.
```

**Comment by slmyldz** [comment]
👍

**Comment by shanshin** [comment]
Fixed in `0.7.0-Alpha`
